roxctl/common/flags/config_file.go

```diff
@@ -0,0 +1,139 @@
+package flags
+
+import (
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/pointers"
+)
+
+type Instance struct {
+	ProfileName       string `json:"name"`
+	CaCertificatePath string `json:"caCertPath"`
+	ApiTokenFilePath  string `json:"apiTokenPath"`
+	Endpoint          string `json:"endpoint"`
+}
```

Comment:
We support other connection parameters, such as:
- `--insecure-skip-tls-verify` and `--insecure`
- `--direct-grpc`
- `--force-http1`
- etc.

I think the config file should support all the same options as the CLI parameters.

Reply:
Ideally, the structure would be runtime or somehow automatically generated so that it is in sync with the CLI connection parameters.